swan-lake/integration-tutorials/transforming-message-formats.md

```diff
@@ -160,7 +160,11 @@ Follow the instructions given in this section to develop the service.
         boolean confirmed;
         string appointmentDate;
     };
+    ```
+
+    Similarly, generate the record types corresponding to the request payload, parameter and return types of the transforming function (e.g. `HealthcareReservation`, `HospitalReservation`). Remove the duplicate `Patient` record if the same type has already been generated.
```

Comment:
Isn't the request payload the same as the parameter of the transforming function?

Anyway, as mentioned previously, saying transforming function here doesn't really make sense to the user. We need to describe these in terms of request/response payloads.

So

```suggestion
    Similarly, generate the record types corresponding to the request payloads to the healthcare service being developed (e.g. `HealthcareReservation`) and the request payload expected by the backend hospital service (e.g.,`HospitalReservation`). The transformation will be done from the former type to the latter. Also remove the duplicate `Patient` record if the same type has already been generated.
```

Reply:
+1. Thanks